src/pages/ChokKahYang/eduList.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./base.scss"
+
+
+export default class EduList extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+
+        const { school, qualified, yearStart, yearEnd } = this.props.qualification
+
+        return (
+            <div className="containerEdu">
```

Comment:
element class name should be kebab-case, but i will ignore this time

Reply:
Noted with thanks.
so for this example will be container-edu?

Reply:
yeap